sdk/compute/Microsoft.Azure.Management.Compute/tests/Microsoft.Azure.Management.Compute.Tests.csproj

```diff
@@ -26,6 +26,12 @@
     </None>
   </ItemGroup>
   
+  <ItemGroup>
+    <None Update="Resources\*.cspkg">
```

Comment:
This is likely the cause of your test failures - you should use '/' as the separator

Reply:
Have changed this, but no luck. This is unlikely to cause the tests to fail as SessionRecords are aslo being copied ( see above) The tests are passing locally, ran them in playback mode as well. Investigating why there is CI failures.